### ISSUE TITLE
Tab thumbnails are incorrectly displayed after a specific scenario (intermitent)

### ISSUE THREAD

**Issue by DianaABarladeanu** [mozilla-mobile/firefox-ios#19770]
**Steps to reproduce**

1. Fresh install FF
1. Open multiple tabs by tapping on the "+" button from tab tray and by tapping on the "+" button from hamburger menu (you need to turn the phone in landscape mode in order to see it).
On iPad, you can open new tabs by tapping on the "+" button located above the toolbar.
Go to tab tray and observe the thumbnails.

**Expected behavior**
All website thumbnails are correctly displayed, with website images/screenshots.

**Actual behavior**
Some websites show placeholder letters instead of thumbnail images/screenshots.

**Device & build information**

* Device: iPhone 15 (17.4.1), iPad Air 3rd gen (17.3)
* Build version: v125 (40715)
* First seen version: ? 

Notes
Attachments: 

https://github.com/mozilla-mobile/firefox-ios/assets/151131988/a9ceb003-e5b1-443b-9bab-d3435770367b

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-8950)

**Comment by data-sync-user** [comment]
➤ Nishant Bhasin commented:

Sophie Amin is this something related to your ticket regarding tab refactor?

**Comment by data-sync-user** [comment]
➤ Sophie Amin commented:

For now I’ve added another call to take a screenshot in the case where we’ve loaded the webpage while it’s still on screen. After discussing with Nish it seems like there was some code that was removed at some point that allowed the webview to load in the background even if the tab was offscreen, and then update the screenshot when the tab was done loading. Because I couldn’t find anywhere in the codebase that seemed to do this currently, I created https://mozilla-hub.atlassian.net/browse/FXIOS-9061 ( https://mozilla-hub.atlassian.net/browse/FXIOS-9061|smart-link ) 

to add this functionality, since I couldn’t find a way to do it quickly while fixing this issue.
